apps/sms/services/js/conversation/conversation_service.js

```diff
@@ -1,11 +1,49 @@
-/*global BridgeServiceMixin */
+/*global bridge,
+         BridgeServiceMixin,
+         BroadcastChannel,
+         Drafts,
+         streamClient,
+         Thread
+*/
 'use strict';
 
-if (!('BridgeServiceMixin' in self)) {
-  importScripts('/services/js/bridge_service_mixin.js');
-}
+/**
+ * Type description for the ConversationSummary object that is return from the
+ * Conversation service.
+ * @typedef {Object} ConversationSummary
+ * @property {number} id Unique identifier of the the conversation.
+ * @property {string} body Short content of the last message in the
+ * conversation.
+ * @property {Array.<string>} participants List of the conversation
+ * participants.
+ * @property {number} timestamp Conversation timestamp.
+ * @property {number} unreadCount Number of conversation messages that user has
+ * not read yet.
+ * @property {string} lastMessageSubject Subject of the last message in the
+ * conversation if it has one.
+ * @property {string} lastMessageType Type of the last message in the
+ * conversation. Can be either 'sms' or 'mms'.
+ * @property {Draft} draft If conversation is draft or has draft, then this
```

Comment:
Yeah, I think it will replace ```Thread``` + ```participants``` field will be converted from ```Array<string>``` to smth like ```Array<{number: string, contactName: string}>``` once we resolve contact names in Conversation service. Not sure about the name though, but it's just small detail :)